src/main/resources/fileTemplates/j2ee/bukkit/Bukkit build.gradle.ft

```diff
@@ -1,14 +1,13 @@
-import org.apache.tools.ant.filters.ReplaceTokens
-
 plugins {
     id 'java'
 }
 
 group = '${GROUP_ID}'
 version = '${PLUGIN_VERSION}'
 
-sourceCompatibility = '1.8'
-targetCompatibility = '1.8'
+java {
+    toolchain.languageVersion = JavaLanguageVersion.of(8)
```

Comment:
Would rather not use `sourceCompatibility` and `targetCompatibility` for Java 9+. I'm not concerned about VSCode compatibility for two reasons:

 1. This is an IntelliJ plugin
 2. We should always focus on a proper Gradle setup, rather than doing not great things for the sake of tools which don't support Gradle correctly.

In short, that sounds like a VSCode problem, not our problem.

Reply:
That's fair, though for 1 -- while this is an intellij plugin, whoever creates the project using the mc dev templates is not necessarily the only contributor, so buildscripts should be reasonably consumable by any common IDE. I think you could reasonably say that VS Code is not a common Java IDE though :p

Reply:
> This will, by default, forcibly download a JDK 8 distribution rather than using any newer version available on the user's computer

Gradle will first search for existing installations (I expect most plugin/mod developer to have installed a JDK 8 by standard means.) For the ones that care about that stuff they can even tell Gradle about their custom installations: https://docs.gradle.org/current/userguide/toolchains.html#sec:custom_loc

It all depends if you want the most reproducible builds possible (in that case we would also set a vendor and implementation.)

I guess your solution is the best for the average developer experience.